app/archivesspace/archivesspace.controller.js

```diff
@@ -112,23 +111,36 @@
         },
       };
 
-      $scope.on_toggle = function(node, expanded) {
-        if (!expanded || !node.has_children || node.children_fetched) {
-          return;
-        }
-
+      // Loads the children of the specified element,
+      // along with arranged SIP contents that might exist
+      var load_element_children = function(node) {
         var on_failure = function(error) {
           Alert.alerts.push({
             type: 'danger',
             message: 'Unable to fetch record ' + node.id + ' from ArchivesSpace!',
           });
-          console.log(Alert);
         };
 
+        node.children = [];
+
         ArchivesSpace.get_children(node.id).then(function(children) {
-          node.children = node.children.concat(children);
+          node.children = node.children.concat(resource.children);
```

Comment:
Looks like it came from when I rebased on top of fcc328da, and I accidentally reverted that line to the pre-fcc328da version during the rebase.